src/themes/default/components/core/blocks/Checkout/OrderReview.vue

```diff
@@ -63,13 +63,16 @@
       <div class="col-xs-12 col-sm-9 col-md-11">
         <div class="row">
           <div class="col-xs-12 col-md-8 px20">
-            <button-full
-              @click.native="placeOrder"
-              data-testid="orderReviewSubmit"
-              :class="{ 'button-disabled' : $v.orderReview.$invalid }"
-            >
-              {{ $t('Place the order') }}
-            </button-full>
+            <slot name="placeOrderButton">
```

Comment:
\+ @pkarw Updated pull with slot and additional class instead global event.
Extension pull https://github.com/develodesign/vsf-payment-paypal/pull/42/files
\+ @collymore